src/main/java/racinggame/model/position/Position.java

```diff
@@ -0,0 +1,53 @@
+package racinggame.model.position;
+
+import java.util.Objects;
+
+public class Position {
+    private int position;
+
+    private static final int DEFAULT_POSITION = 0;
+    private static final int PLUS_POSITION = 1;
+
+    public Position() {
+        this.position = DEFAULT_POSITION;
```

Comment:
```suggestion
       this(DEFAULT_POSITION);
```
주 생성자를 호출해 구현할 것을 추천